tools.md

```diff
@@ -12,7 +12,7 @@ The following table is an alphabetical list of known commands defined in WordPre
 | amt                   | [Add-Meta-Tags command line interface](http://www.codetrax.org/projects/wp-add-meta-tags/wiki/Command_Line_Interface)
 | any-ipsum             | [Any Ipsum](https://wordpress.org/plugins/any-ipsum/)
 | backup                | [BackUpWordPress](http://wordpress.org/extend/plugins/backupwordpress/)
-| backupbuddy           | [BackupBuddy](http://getbackupbuddy.com/)
+| backupbuddy           | [BackupBuddy](https://ithemes.com/purchase/backupbuddy/)
```

Comment:
Don't change this one, the link still works and is used for indirection.

Reply:
Thank you @schlessera , I have restored **Backupbuddy** URL. And please let me know if any thing is worung.

Thanks.